core/templates/dev/head/pages/exploration-player-page/exploration-player-page.mainpage.html

```diff
@@ -44,10 +44,9 @@
         </page-footer>
       </base-content>
     </div>
+    @require('../footer_js_libs.html')
     <script src="/third_party/static/ckeditor-4.9.2/ckeditor.js"></script>
-    <script src="/templates/dev/head/mathjaxConfig.js"></script>
```

Comment:
are we expecting this change to be done in all the places where we have imported `mathjaxConfig`?

Reply:
Yes.
Done. Thanks for pointing this out.